### PR TITLE
Only add label class if label is visible

### PR DESCRIPTION
**Pull request by maxbeizer**
Closes https://github.com/primer/view_components/issues/1140

This will only add the label classes if the label is visible. How's that look to y'all? 🙇

### CODE REVIEW

File: app/components/primer/beta/auto_complete/auto_complete.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%= render Primer::BaseComponent.new(**@system_arguments) do %>
-  <label for="<%= @input_id %>" class="<%= @label_classes %>">
+  <label for="<%= @input_id %>" class="<%= @label_classes if @is_label_visible %>">
```
Comment (camertron) [inline app/components/primer/beta/auto_complete/auto_complete.html.erb:2]: This should work just fine, but what about setting `@label_classes` to `nil` in the Ruby code?

**Review by camertron — APPROVED** [review]

File: app/components/primer/beta/auto_complete/auto_complete.html.erb (hunk 1 above)
Comment (maxbeizer) [reply app/components/primer/beta/auto_complete/auto_complete.html.erb:2]: So, I looked at that 😄 .

Currently label class is set in a ternary, so adding another conditional would be a bigger code change. If you think it's clearer that way I'm happy to update it 👍

**Review by camertron — APPROVED** [review]

**Comment by maxbeizer** [comment]
Thanks @jonrohan!

**Comment by camertron** [comment]
Sorry @maxbeizer, looks like @jonrohan got you sorted :)

**Pull request merged**